### PR TITLE
Adding ability to delete a specified key from Redis. Adding tests

### PR DESCRIPTION
**Pull request by willvelida**
This PR contains the following changes:

- Adds the ```Delete()``` method to ```IRedisCache.cs``` interface. This method is implemented in ```RedisDb.cs```. The purpose of this method is to remove a specified key from the cache.
- Adds a test called ```DeleteKey_SuccessfullyRemovesKey``` to validate that the method removes the specified key from the cache.
- Adds a test called ```DeleteKey_IgnoresKeyWhenItDoesNotExist``` to validate that the key is ignored when it does not exist in the cache as specified [here](https://redis.io/commands/del).
- Updates ```README.md``` to include the new ```Delete()``` method.

### CODE REVIEW

File: src/AzureCacheRedisClientTests/RedisDbTests.cs [hunk 1]
```diff
@@ -56,6 +56,38 @@ public async Task UsageTest()
             Assert.AreEqual(item, cachedItem);
         }
 
+        [TestMethod]
```
Comment (DanielLarsenNZ) [inline src/AzureCacheRedisClientTests/RedisDbTests.cs:63]: Please add `[TestCategory("Integration")]`

**Review by DanielLarsenNZ — CHANGES_REQUESTED** [review]

File: src/AzureCacheRedisClientTests/RedisDbTests.cs (hunk 1 above)
Comment (DanielLarsenNZ) [reply src/AzureCacheRedisClientTests/RedisDbTests.cs:63]: @willvelida, you may need to merge the Test project changes I have made on main for the tests to work (I have just been fixing them)

Comment (willvelida) [reply src/AzureCacheRedisClientTests/RedisDbTests.cs:63]: TestCategory annotation has been applied to both `DeleteKey_SuccessfullyRemovesKey ` and `DeleteKey_IgnoresKeyWhenItDoesNotExist `

**Review by DanielLarsenNZ — APPROVED** [review]
:shipit:

**Comment by DanielLarsenNZ** [comment]
Thanks for your contribution @willvelida 🎉

**Pull request merged**